cmd/ocic/container.go

```diff
@@ -10,10 +10,10 @@ import (
 
 	"github.com/urfave/cli"
 	"golang.org/x/net/context"
+	remocommandconsts "k8s.io/apimachinery/pkg/util/remotecommand"
```

Comment:
Nit, and don't hold up the submit for this, but could you go back and change this variable name to "remotecommandconsts"?  (Add 'te' after 'remo')